### DISCUSSION TITLE
Thoughts on 1.4's new code?

### DISCUSSION THREAD

**Discussion by Toriality** [Toriality/DYOM-Bingo#20] [Q&A] [answered]
In the last few days I have been refactoring the whole site code, to make it more organized and divided into small components. With that concluded, I realized some points that can be positive and negative.

As a positive point, I would point out that the code is more organized and readable. Also, the code is more scalable and new complex features can be included without too much difficulty.

On the other hand, the model seems to be adding an extra level of boilerplate and a lot of new files.

[Check the branch](https://github.com/Toriality/DYOM-Bingo/tree/1.4)

**I would love to get your feedback and open this new discussion about the benefits and problems encountered in the new model compared to the current model.**

**Comment by laszlostark** [comment, marked as answer]
I like the new structure from the refactor. I'm not all that familiar with the DYOM-Bingo codebase myself but having the script in smaller packets certainly makes it easier to digest.
